configure

```diff
@@ -133,6 +133,11 @@ parser.add_option('--openssl-fips',
     dest='openssl_fips',
     help='Build OpenSSL using FIPS canister .o file in supplied folder')
 
+parser.add_option('--openssl-use-def-ca-store',
+    action='store_true',
+    dest='use_openssl_ca_store',
+    help='use OpenSSL supplied Root CA store instead of bundled copy')
```

Comment:
There is lots of inconsistency in the capitalization and formatting of the configure messages, but it looks like the style that is most readable would be:

```
Use OpenSSL supplied Root CA store instead of compiled-in Mozilla CA store.
```

^--- Makes it a capitalized sentence, and makes it clear that the "bundled copy" is not a copy of the "OpenSSL supplied Root CA store".